src/macros.rs

```diff
@@ -14,52 +14,46 @@
 
 #![allow(unused_macros)]
 
-macro_rules! task_log {
-    ($lvl:expr, $($args:tt)+) => {{
-        use crate::CONTEXT;
+// $ctx in all macro arguments is of type crate::ConnectionContext
 
-        CONTEXT.with(|m| {
-            if let Some(ref ctx) = m.borrow().as_ref() {
-                log!(ctx.logger.as_ref(), $lvl, "", $($args)+)
-            } else {
-                log!(crate::DEFAULT_LOGGER, $lvl, "", $($args)+)
-            }
-        });
+macro_rules! task_log {
```

Comment:
This doesn't seem to be very useful now that the context is no longer a thread local. As a followup (after this PR), could you rewrite all uses of it to the regular log macros?

Reply:
I agree thats a good change, mildly confusing how log macros take `ctx.logger.as_ref()` but the task_log just take `ctx`